presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java

```diff
@@ -37,27 +33,19 @@
 import static com.facebook.presto.tests.statistics.MetricComparison.Result.MATCH;
 import static com.facebook.presto.tests.statistics.MetricComparison.Result.NO_BASELINE;
 import static com.facebook.presto.tests.statistics.MetricComparison.Result.NO_ESTIMATE;
-import static com.facebook.presto.tests.tpch.TpchQueryRunner.createQueryRunnerWithoutCatalogs;
 import static java.lang.String.format;
 import static java.util.Collections.emptyList;
-import static java.util.Collections.emptyMap;
 import static java.util.stream.Collectors.groupingBy;
 import static org.testng.Assert.assertEquals;
 
 public class TestTpchDistributedStats
+        extends AbstractTestDistributedStats
 {
     public static final int NUMBER_OF_TPCH_QUERIES = 22;
```

Comment:
I disagree, it's that knowledge that should be moved out of the abstract class :)